guides/common/assembly_administering-hosts.adoc

```diff
@@ -22,6 +22,26 @@ endif::[]
 
 include::modules/proc_changing-the-environment-of-a-host.adoc[leveloffset=+1]
 
+include::modules/con_managing-multiple-content-view-environments.adoc[leveloffset=+1]
```

Comment:
I do not know this subject well but it seems to me that managing multiple content views could have its own assembly, in the Managing Content guide. I'm not sure a user would think to look here, especially since the Content guide has info on content views.

Reply:
In the previous section, "Changing the environment of a host", we need to add a note anyway that says something like

```
If you assign a single content view environment in the web UI here, and your host was previously a multi-environment host, the multiple content view environments will be replaced by the single environment you set here.
```

We could then add
```
For more information about assigning multiple content view environments to hosts, see xxxxx
```

and link to the Managing Content guide.

Reply:
As to placement, I would recommend a new chapter directly before "Synchronizing content between Foreman servers." (That way it's after the CV and LCE chapters.)